src/main/java/org/lwjglx/openal/AL.java

```diff
@@ -41,6 +43,17 @@ public static void create(String deviceArguments, int contextFrequency, int cont
         attribs.put(org.lwjgl.openal.ALC10.ALC_SYNC);
         attribs.put(contextSynchronized ? org.lwjgl.openal.ALC10.ALC_TRUE : org.lwjgl.openal.ALC10.ALC_FALSE);
 
+        /////////////////////////////////////////////
+        // HRTF
+        if (!OPENAL_ENABLE_HRTF) {
```

Comment:
I'd rather this be `Config.OPENAL_ENABLE_HRTF` without the static import

Reply:
I can change it